samples/grids/grid/editing-excel-style-custom/src/index.ts

```diff
@@ -0,0 +1,122 @@
+import 'igniteui-webcomponents-grids/grids/combined';
+import { ComponentRenderer, WebGridDescriptionModule } from 'igniteui-webcomponents-core';
+import { IgcCellType, IgcGridComponent } from 'igniteui-webcomponents-grids/grids';
+import { NwindData } from './NwindData';
+
+import "igniteui-webcomponents-grids/grids/themes/light/bootstrap.css";
+
+import "./index.css";
+
+export class Sample {
+
+    private grid1: IgcGridComponent
+    private _bind: () => void;
+    private shouldAppendValue = false;
+
+    constructor() {
+        var grid1 = this.grid1 = document.getElementById('grid1') as IgcGridComponent;
+    
+        this._bind = () => {
+            grid1.data = this.nwindData;
+
+            grid1.addEventListener('activeNodeChange', (event: any) => {
+                grid1.endEdit();
+                (grid1.getElementsByClassName("igx-grid__tbody-content")[0] as any).focus();
+
+            });
+            grid1.addEventListener('keydown', (event: KeyboardEvent) => {
+                var code = event.code;
+                var activeElem = grid1.selectedCells[0];
+
+                if ((event.code >= 'Digit0' && event.code <= 'Digit9') || 
+                    (event.code >= 'KeyA' && event.code <= 'KeyZ') && 
+                    event.code !== 'Enter') {
+        
+                    if (activeElem && activeElem.editMode === false) {
+                        activeElem.editMode = true;
+                        activeElem.editValue = event.key;
+                        this.shouldAppendValue = true;
+                        grid1.markForCheck();
+                    } else
+                    
+                    if (activeElem && activeElem.editMode && this.shouldAppendValue) {
+                        event.preventDefault();
+                        activeElem.editValue = activeElem.editValue + event.key;
+                        this.shouldAppendValue = false;
+                      }
+                }
+
+                if (code === 'Backspace') {
+                    if(activeElem == null) {
+                        return;
+                    }
+                    const rowIndex = activeElem.row.index;
+                    const columnKey = activeElem.column.field; 
+            
+                    grid1.data[rowIndex][columnKey] = '';
```

Comment:
This clears the value, however it does not trigger change detection right away, so you need to do some action , like click on/navigate to another cell to see the value is cleared. Maybe call `grid1.markForCheck();` afterwards.

Also it might be better to set the value to null/undefined, since empty string might not be valid for a number columns.